gateway-server/src/main/java/org/apache/knox/gateway/GatewayMessages.java

```diff
@@ -34,13 +34,13 @@
 public interface GatewayMessages {
 
   @Message( level = MessageLevel.FATAL, text = "Failed to parse command line: {0}" )
-  void failedToParseCommandLine( @StackTrace( level = MessageLevel.DEBUG ) ParseException e );
+  void failedToParseCommandLine( @StackTrace( level = MessageLevel.FATAL ) ParseException e );
```

Comment:
Is this change necessary? I would have assumed that the MessageLevel was a minimum level to expose not the exact level.

Reply:
The JIRA requested that there should be no FATAL messages displayed on DEBUG level (which I agree with); so I believe this is necessary. Any objection?

Reply:
Ok makes sense. Didn't tie the two pieces together. The stacktrace level is basically opposite of what I expected.